qe_metrics/utils/issue_utils.py

```diff
@@ -126,5 +128,8 @@ def delete_old_issues(days_old: int) -> None:
         lambda _issue: _issue.last_updated < (datetime.now().date() - timedelta(days=days_old))
     )
     LOGGER.info(f"Deleting {len(issues)} issues that haven't been updated in {days_old} days from the database")
-    [issue.delete() for issue in issues]
-    orm.commit()
+    try:
```

Comment:
you do not need this try:except anymore
@ignore_exceptions(logger=LOGGER) will do it for you

Reply:
we need to check if we have more functions that need `@ignore_exceptions`

as I see it, all non-critical functions should not raise, only server start and database init should raise and fail the program, once it's up we should never fail (but we need logger and implement Slack notification)

Reply:
Okay, I will add it to this one, and then we can do a separate PR to add it to the other functions as needed. #83